compiler/testData/diagnostics/tests/platformTypes/nullabilityWarnings/array.fir.kt

```diff
@@ -21,11 +21,11 @@ fun test() {
     val platformJ = J.staticJ
 
     platformNN[0]
-    <!INAPPLICABLE_CANDIDATE!>platformN[0]<!>
+    <!UNSAFE_CALL!>platformN[0]<!>
     platformJ[0]
 
     platformNN[0] = 1
-    <!INAPPLICABLE_CANDIDATE!>platformN[0]  = 1<!>
+    <!UNSAFE_CALL!>platformN[0]  = 1<!>
```

Comment:
For the record. In addition to `.`, this is another place where the diagnostic positioning matters: it should be array access.